src/components/sectionList/listActions/SectionListActions.tsx

```diff
@@ -99,6 +105,24 @@ export const ActionMore = ({
                                 }}
                             ></MenuItem>
                         </TooltipWrapper>
+
+                        <TooltipWrapper
+                            condition={!deletable}
+                            content={TOOLTIPS.noEditAccess}
+                        >
+                            <MenuItem
+                                dense
+                                disabled={!deletable}
+                                label={i18n.t('Delete')}
+                                icon={<IconDelete16 />}
+                                onClick={() => {
+                                    onDeleteClick()
+                                    setOpen(false)
+                                }}
+                                target="_blank"
+                                href={href}
```

Comment:
This is not a link? Should remove target and href?

Reply:
removed